src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -124,7 +124,7 @@ endif()
 
 install (TARGETS System.Globalization.Native-Static DESTINATION ${STATIC_LIB_DESTINATION} COMPONENT libs)
 
-if(NOT CLR_CMAKE_TARGET_APPLE AND NOT CLR_CMAKE_TARGET_ANDROID)
+if(NOT CLR_CMAKE_TARGET_APPLE AND NOT CLR_CMAKE_TARGET_ANDROID AND NOT CLR_CMAKE_TARGET_ALPINE_LINUX)
```

Comment:
It might be worth it to enhance the `verify-so.sh` script to be able to handle cross-build scenarios, though I don't know how easy that would be. This would be a good follow-up issue.

Reply:
The targets excluded here don't need the verification. Linkers on these platforms don't succeed linking executables with undefined symbols.